app/components/Sidebar/components/CollectionLink.tsx

```diff
@@ -81,7 +81,15 @@ function CollectionLink({
       const { id, collectionId } = item;
       if (monitor.didDrop()) return;
       if (!collection) return;
-      if (collection.id === collectionId) return;
+
+      const document = documents.get(id);
+      if (
+        collection.id === collectionId &&
+        document?.parentDocumentId?.length === undefined
```

Comment:
FYI: `parentDocumentId` is types as `string | undefined` but this was actually returning `null` when a parent is not set. Checking `length` seemed like the safest approach.

Reply:
The types are very new – probably needs tweaking :) Either way an empty check should be fine `!document?.parentDocumentId`

Reply:
👍  updated

I think I've been conditioned from getting burned by `0 == false` too many times in the past...  Obviously not a concern here (unless `0` is a valid parent id).